src/components/sidebar.tsx

```
@@ -33,18 +51,65 @@ export default function Sidebar() {
+          {/* Top: Logo and Navigation */}
+          <div>
+            {/* Header */}
+            <div className="flex items-center justify-between h-16 border-b border-orange-500 px-4">
+              <Link
+                href="/dashboard"
+                className="flex items-center gap-2 text-xl font-bold text-white"
+              >
+                <Store className="w-6 h-6" />
+                <span>STORE</span>
+              </Link>
+              <button onClick={() => setSidebarOpen(false)} className="md:hidden text-2xl">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)